dataactbroker/handlers/userHandler.py

```diff
@@ -58,6 +58,13 @@ def deleteToken(self,token):
         self.session.delete(oldToken)
         self.session.commit()
 
+    def getUsers(self, agency=None):
+        """ Return all users in the database """
+        query = self.session.query(User)
+        if agency is not None:
+            return query.filter(User.agency == agency)
+        return query.order_by(User.user_status_id).all()
```

Comment:
If the line inside changes to query = query.filter(...), suddenly this function makes a lot more sense.